### PR TITLE
Api for Employee Controller

### PR DESCRIPTION
**Pull request by ror-uday**

### CODE REVIEW

File: app/controllers/api/v1/employees_controller.rb [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+  module V1
+    class EmployeesController < ApplicationController
+      # before_action :authenticate_employee!
+      # load_and_authorize_resource
+      before_action :set_employee, only: [:update]
+      protect_from_forgery with: :null_session
+
+      # GET /employees/1 or /employees/1.json
+      def show
+        @employee = get_employee
```
Comment (rorpyll) [inline app/controllers/api/v1/employees_controller.rb:13]: why we need instance variable?

**Review by rorpyll — CHANGES_REQUESTED** [review]

File: app/controllers/api/v1/employees_controller.rb (hunk 1 above)
Comment (ror-uday) [reply app/controllers/api/v1/employees_controller.rb]: change

File: app/serializers/v1/employee_serializer.rb [hunk 2]
```diff
@@ -0,0 +1,14 @@
+# frozen_string_literal: true
+
+# module Api::V1
+  class EmployeeSerializer < ActiveModel::Serializer
+    attributes :id, :name, :email, :father_name, :mother_name, :phone_number, :address, :image, :leave_count,
+               :department, :bank_name, :account_number, :pan_card_number, :aadhar_card_number, :salary, :primary_skill, :date_of_birth, :role_id, :designation
+
+    def designation
```
Comment (rorpyll) [inline app/serializers/v1/employee_serializer.rb:8]: def designation
        object.designation.name
    end

Comment (rorpyll) [inline app/serializers/v1/employee_serializer.rb:8]: def designation
        object.designation.name
    end

Comment (ror-uday) [reply app/serializers/v1/employee_serializer.rb:8]: done

**Pull request closed without merging**